codegens/csharp-restsharp/test/unit/convert.test.js

```diff
@@ -257,6 +258,33 @@ describe('csharp restsharp function', function () {
       sanitizedOptions = sanitizeOptions(testOptions, getOptions());
       expect(sanitizedOptions).to.deep.equal(testOptions);
     });
+
+    it('should use client.UserAgent instead of AddHeader function', function () {
+      var request = new sdk.Request({
```

Comment:
Here you can define the object and use it, don't need a file.

Reply:
However, I got a lint error: `error  Line 262 exceeds the maximum line length of 120   max-len`, the test UA exceed the max line length allowed. Any suggestions on how to solve it ? Thank you!